### ISSUE TITLE
Seems broken on latest Node.js v14.2.0

### ISSUE THREAD

**Issue by kuvam** [Bannerets/tdl#80]
Error log is below. This project ran successfully before upgrading my Node.js version. I'm running Windows 8.1 64-bit.

```
D:\Code\tdl1>node -v
v14.2.0

D:\Code\tdl1>node index


#
# Fatal error in , line 0
# Check failed: result.second.
#
#
#
#FailureMessage Object: 0000005494D7B9D0
 1: 00007FF63884CEDF v8::internal::wasm::DisjointAllocationPool::~DisjointAllocationPool+74383
 2: 00007FF638787BEF std::basic_ostream<char,std::char_traits<char> >::operator<<+55439
 3: 00007FF63934E432 V8_Fatal+162
 4: 00007FF638E173AD v8::internal::BackingStore::Reallocate+653
 5: 00007FF639042B09 v8::ArrayBuffer::GetBackingStore+137
 6: 00007FF638812F98 v8::internal::Malloced::operator delete+15784
 7: 00007FF6388179F5 node::Buffer::Length+1813
 8: 00007FF6388180B3 node::Buffer::New+163
 9: 00007FF63881D886 napi_create_external_buffer+230
10: 00007FFABABE8158 Napi::Buffer<char>::New<<lambda_efb3f083e984881c8a8dc51f31353307> >+120 [d:\code\tdl1\node_modules\node-addon-api\napi-inl.h]:L1824
11: 00007FFABABEC33E FFI::FFI::InitializeBindings+5118 [d:\code\tdl1\node_modules\ffi-napi\src\ffi.cc]:L117
12: 00007FFABABEADA9 Init+73 [d:\code\tdl1\node_modules\ffi-napi\src\ffi.cc]:L304
13: 00007FFABABE851F Napi::details::WrapCallback<<lambda_8c7377a71108815b2eee34f099f4cb99> >+63 [d:\code\tdl1\node_modules\node-addon-api\napi-inl.h]:L104
14: 00007FFABABE7301 __napi_Init+81 [d:\code\tdl1\node_modules\ffi-napi\src\ffi.cc]:L309
15: 00007FF63881CD0B node_module_register+5691
16: 00007FF63881A0C3 node::Buffer::New+7427
17: 00007FF63881A982 node::Buffer::New+9666
18: 00007FF639005E3F v8::internal::Builtins::builtin_handle+330191
19: 00007FF6390053CE v8::internal::Builtins::builtin_handle+327518
20: 00007FF6390056C7 v8::internal::Builtins::builtin_handle+328279
21: 00007FF639005513 v8::internal::Builtins::builtin_handle+327843
22: 00007FF6390E356D v8::internal::SetupIsolateDelegate::SetupHeap+466093
23: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
24: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
25: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
26: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
27: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
28: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
29: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
30: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
31: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
32: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
33: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
34: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
35: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
36: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
37: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
38: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
39: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
40: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
41: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
42: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
43: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
44: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
45: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
46: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
47: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
48: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
49: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
50: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
51: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
52: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
53: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
54: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
55: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
56: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
57: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
58: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
59: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
60: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
61: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
62: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
63: 00007FF63907B708 v8::internal::SetupIsolateDelegate::SetupHeap+40520
64: 00007FF6390792EE v8::internal::SetupIsolateDelegate::SetupHeap+31278
65: 00007FF639078EDC v8::internal::SetupIsolateDelegate::SetupHeap+30236
66: 00007FF638F5CDEF v8::internal::Execution::CallWasm+1743
67: 00007FF638F5C5FF v8::internal::Execution::Call+191
68: 00007FF6390361A7 v8::Function::Call+615
69: 00007FF63881EFFF napi_unref_threadsafe_function+2543
70: 00007FF638822B97 node::Start+1159
71: 00007FF638822F91 node::Start+2177
72: 00007FF63886BF38 node::LoadEnvironment+56
73: 00007FF6387AFBE3 EVP_CIPHER_CTX_buf_noconst+39939
74: 00007FF638822830 node::Start+288
75: 00007FF6386866FC RC4_options+339452
76: 00007FF63959782C v8::internal::compiler::RepresentationChanger::Uint32OverflowOperatorFor+156812
77: 00007FFB021213D2 BaseThreadInitThunk+34
78: 00007FFB02E154F4 RtlUserThreadStart+52

D:\Code\tdl1>
```

**Comment by Bannerets** [comment]
https://github.com/node-ffi-napi/node-ffi-napi/issues/71

**Comment by Bannerets** [comment]
Fixed in tdl-tdlib-ffi v2.0.0

**Comment by kuvam** [comment]
Thanks for the update.
